Common/util/string_utils.cpp

```diff
@@ -203,5 +204,12 @@ void StrUtil::WriteString(const String &s, Stream *out)
         out->Write(s.GetCStr(), len);
 }
 
+void StrUtil::WriteCStr(const String &s, Stream *out, int max_len)
+{
+    max_len = Math::Min(max_len, s.GetLength());
```

Comment:
Well, I just found a bug in this function... parameter's default value -1 will make nothing to be written, instead of whole string, as was probably planned.